packages/components/src/Tag/Tag.stories.tsx

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import * as React from "react";
+import Tag, { stylesByColor } from "./Tag";
+import type { Color } from "./Tag";
+import type { Story } from "@storybook/react";
+import WarningIcon from "../SVGIcon/Icons/Warning";
+import styles from "./Tag.stories.module.css";
+
+const colorOptions: Array<Color> = Object.keys(stylesByColor);
+
+export default {
+  title: "Tag",
+  component: Tag,
+
+  argTypes: {
+    color: {
+      control: {
+        type: "select",
+        options: colorOptions,
+      },
+    },
+  },
+};
+
+type Args = React.ElementProps<typeof Tag>;
+
+const Template: Story<Args> = (args) => <Tag {...args} />;
+
+const defaultArgs = {
+  children: "Tag text",
+  color: "warning",
+};
+
+export const Default = Template.bind(null);
+Default.args = {
+  ...defaultArgs,
+};
+
+export const ColorVariants: Story<Args> = (args) => (
+  <div className={styles.tagList}>
+    {colorOptions.map((color) => {
+      return <Tag key={color} {...args} color={color} />;
+    })}
+  </div>
+);
+ColorVariants.args = {
+  ...defaultArgs,
+};
```

Comment:
oop just noticed -- could we add the OutlineVariants story as well? 
```
export const OutlineVariants: Story<Args> = args => (
  <div className={styles.tagList}>
    {colorOptions.map(color => {
      return <Tag key={color} {...args} color={color} variant="outline" />;
    })}
  </div>
);
OutlineVariants.args = {
  ...defaultArgs,
};
```

Reply:
Oh yeah! I tried to get the recent changes but I must've missed some things. I'll double-check the css as well to make sure it's synced with the traject version.